config/vufind/config.ini

```
@@ -2668,4 +2668,4 @@ description = "The REST API provides access to search functions and records cont
 [HTML_Rendering_Contexts]
 ;allowed_elements[default] = ""
 ;allowed_elements[heading] = "a,b,br,em,i,span,strong,sub,sup,u"
-;allowed_elements[link] = "abbr,acronym,b,bdo,big,br,cite,dfn,em,i,img,q,samp,small,span,strong,sub,sup,var"
+;allowed_elements[link] = "abbr,acronym,b,bdo,big,br,cite,dfn,em,i,img,q,samp,small,span,strong,sub,sup,var"
```

The trailing line break seems to have been accidentally removed from this file; we should put it back to prevent an unnecessary diff in the PR.

```suggestion
;allowed_elements[link] = "abbr,acronym,b,bdo,big,br,cite,dfn,em,i,img,q,samp,small,span,strong,sub,sup,var"

```